packages/phoenix-event-display/src/tests/managers/loading-manager.test.ts

```diff
@@ -0,0 +1,54 @@
+import { LoadingManager } from '../../managers/loading-manager';
+
+describe('LoadingManager', () => {
+  let loadingManager: LoadingManager;
+
+  beforeEach(() => {
+    loadingManager = new LoadingManager();
+  });
+
+  afterEach(() => {
+    loadingManager.reset();
+  });
+
+  it('should create an instance', () => {
+    expect(loadingManager).toBeTruthy();
+  });
+
+  it('should add an item which is to be loaded', () => {
+    loadingManager.addLoadableItem('item');
+    expect(loadingManager.toLoad.length).toBe(1);
+  });
+
+  it('should call the function when an item has finished loading', () => {
+    jest.spyOn(loadingManager, 'itemLoaded');
+    loadingManager.addLoadableItem('item');
+    loadingManager.itemLoaded('item');
+    expect(loadingManager.itemLoaded).toHaveBeenCalled();
+  });
+
+  it('should call the function when loading of an item progresses', () => {
+    jest.spyOn(loadingManager, 'onProgress');
+    loadingManager.addLoadableItem('item');
+    loadingManager.onProgress('item', 50);
+    expect(loadingManager.onProgress).toHaveBeenCalled();
+  });
+
+  it('should add a listener for when all items have loaded', () => {
+    jest.spyOn(loadingManager, 'addLoadListener');
+    loadingManager.addLoadListener(() => {});
+    expect(loadingManager.addLoadListener).toHaveBeenCalled();
```

Comment:
All of the remaining test cases in this suite including this one shouldn't be needed. If you are testing the behavior you check if the listeners were correctly called when the loading of all items was completed. Which I suggested in my previous comments.